### PR TITLE
Fix a few typos, etc. in the documentation

### PR DESCRIPTION
**Pull request by mulhod**

### CODE REVIEW

**Review by desilinguist — APPROVED** [review]
Thank you for fixing these!

**Review by desilinguist — APPROVED** [review]

File: doc/run_experiment.rst [hunk 1]
```diff
@@ -350,8 +350,9 @@ Regressors:
     *   **Ridge**: `Ridge Regression <https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Ridge.html#sklearn.linear_model.Ridge>`__
     *   **SGDRegressor**: `Stochastic Gradient Descent Regression <https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDRegressor.html>`__
     *   **SVR**: `Support Vector Regression using LibSVM <https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVR.html#sklearn.svm.SVR>`__
+<<<<<<< HEAD
```
Comment (Frost45) [inline doc/run_experiment.rst:353]: This line should probably be removed

File: doc/run_experiment.rst [hunk 2]
```diff
@@ -999,9 +1001,10 @@ SKLL provides the following metrics but you can also write your own :ref:`custom
...
-    *   **neg_mean_squared_error**: The negative of the `mean squared error <https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html>`__ regression loss. Since scikit-learn `recommends <https://scikit-learn.org/stable/modules/model_evaluation.html#common-cases-predefined-values>`__ using negated loss functions as scorer functions, SKLL does the same for the sake of consistency.
-    *   **neg_root_mean_squared_error**: The negative of the `mean squared error <https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html>`__ regression loss, with ``squared`` set to False. Since scikit-learn `recommends <https://scikit-learn.org/stable/modules/model_evaluation.html#common-cases-predefined-values>`__ using negated loss functions as scorer functions, SKLL does the same for the sake of consistency.
+<<<<<<< HEAD
```
Comment (Frost45) [inline doc/run_experiment.rst:1004]: And this too

**Review by Frost45 — APPROVED** [review]
Some minor suggestions!

File: doc/run_experiment.rst (hunk 1 above)
Comment (desilinguist) [reply doc/run_experiment.rst:353]: Good catch!! I don't know how I forgot those in there!!

**Pull request merged**